.bldr.toml

```diff
@@ -541,6 +541,11 @@ plan_path = "mg"
 plan_path = "mongo-tools"
 [mongodb]
 plan_path = "mongodb"
+[mosquitto]
+plan_path = "mosquitto"
+paths = [
```

Comment:
For this package I don't think you should actually need the `paths = [ "mosquitto/*" ]` option. In the core-plans repo this is mostly for planfiles that are sourcing other planfiles.

Reply:
Done ;)